packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -478,7 +472,7 @@ export const ChatScreen = () => {
   }, [])
 
   const handleOnContentSizeChanged = useCallback(
-    (_contentWidth, contentHeight) => {
+    (contentWidth, contentHeight) => {
```

Comment:
won't this break linter?

Reply:
seems like it passed.. i've had unused vars without the _ before.

Reply:
seems like a bug with our linter 🤷

Reply:
ha fair